patches/server/0732-Hide-unnecessary-itemmeta-from-clients.patch

```diff
@@ -107,6 +111,8 @@ index a5cf00a41c61cc9f851781cc4f55c9bb47c2f5b6..f306c9e8f46d80d4c4fdc671e3754b3f
 +                // the position and fake the dimension
 +                tag.putString("LodestoneDimension", "paper:paper");
 +            }
++            // The id of a filled map
++            tag.remove("map");
```

Comment:
What does this visually change?

Reply:
Vanilla does not have any visual changes, but [some mods](https://modrinth.com/mod/not-enough-animations) which aren't necessarily cheats could behave differently.